3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/apirequest/ApiRequestIntegrationTest.scala

```diff
@@ -0,0 +1,275 @@
+/*
+ * Copyright (c) 2012-2016 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.enrich.common
+package enrichments
+package registry
+package apirequest
+
+// json4s
+import org.json4s.JsonAST._
+import org.json4s.jackson.{ parseJson, prettyJson }
+
+// specs2
+import org.specs2.Specification
+import org.specs2.scalaz.ValidationMatchers
+import org.specs2.matcher.Matcher
+
+// Iglu
+import com.snowplowanalytics.iglu.client.SchemaKey
+
+// This project
+import outputs.EnrichedEvent
+import utils.JsonUtils
+
+object ApiRequestEnrichmentIntegrationTest {
+  def continuousIntegration: Boolean = sys.env.get("CI") match {
+    case Some("true") => true
+    case _ => false
+  }
```

Comment:
Well, I'm more concerned not about quickness, but about a case where developer can just forget to set CI=true and miss some important failure, because it was skipped silently. So, yes I think it is a good idea. I'll create one.